lib/beaker/dsl/structure.rb

```diff
@@ -1,4 +1,5 @@
 require 'beaker/dsl/assertions'
+require 'readline'
```

Comment:
@Andrew-Hayes we don't want stray `require` calls including code that oftentimes won't be used on a particular run.

Would you move this require inside the `manual_step` method?